apps/frontend/src/libs/components/input/input.tsx

```diff
@@ -26,6 +27,7 @@ type Properties<T extends FieldValues> = {
 const Input = <T extends FieldValues>({
 	control,
 	errors,
+	hasVisuallyHiddenLabel,
```

Comment:
maybe just isVisuallyHidden or isHidden?

Reply:
I think isVisuallyHidden or isHidden we apply for the Input component. 
But I need to hide the Icon Label